modules/google-analytics/languages/wp-google-analytics.pot

```diff
@@ -0,0 +1,185 @@
+# Copyright (C) 2012 WP Google Analytics
```

Comment:
We'll need to remove this from the PR, as we rely on language packs to serve translations on WordPress.org.